README.md

```diff
@@ -8,14 +8,52 @@ Coordinate services for YUL-DC project
 
 - Download [Docker Desktop](https://www.docker.com/products/docker-desktop) and log in
 
-## Starting the services
 
-- Start all the services locally for development
+## Install
+
+Download the yul-dc-camerata repo and install the gem.
+
+```bash
+git clone git@github.com:yalelibrary/yul-dc-camerata.git
+cd yul-dc-camerata
+rake install
+```
+
+## Updates
+
+You can get the latest version at any point by updating the code and reinstalling
+
+```bash
+cd yul-dc-camerata
+git pull origin master
+rake install
+```
+
+## General Use
+
+Once camerata is installed on your system, interactions happen through the 
+camerata command-line tool or through its alias `cam`.  The camerata tool can be 
+used to bring the development stack up and down locally, interact with the 
+docker containers, deploy, run the smoke tests and otherwise do development 
+tasks common to the various applications in the yul-dc application stack.
+
+All buildin commands can be listed with `cam help` and individual usage 
+information is available with `cam help COMMAND`.  Please note that deployment 
+commands (found in the `./bin` directory) are pass through and are therefor not 
+listed by the help command.  See th usage for those below. 
+
+To start the application stack, run `cam up`. This is the equivalent of running 
+`docker-compose up blacklight`. This starts all of the applications as they are 
+all dependencies of yul-blacklight. Camerata is smart. If you start `cam up` from 
+a blacklight code check out it will mount that code for local development 
```

Comment:
Does this effectively mean I should be able to `cam up` from the blacklight repo? Wasn't able to do that but I could be off in my interpretation here. I ask bc I know it's a goal to reduce the number of ways we `up` our apps in development and thought this was where we were going to do that.

Reply:
We did mention a possible teach out for this shift. Perhaps that is the place for this question.